### ISSUE TITLE
Dual Tone Icons: Contrast Ration Issue

### ISSUE THREAD

**Issue by BeckerWdf** [eclipse-platform/ui-best-practices#145]
When looking at 

<img width="443" height="242" alt="Image" src="https://github.com/user-attachments/assets/fb6bb037-6e46-4ae3-9e01-a336fa587fe0" />

I would say that the contrast ration of the "disconnect" icon (5th icon from the left in the toolbar) is too low in the dark theme. (I think it should be 3:1 )
Can we choose a grey color that has good contrast in light and in dark theme? Or do we anyway expect that once we use the icons we will switch colors based on the theme (the background color)?

**Comment by Michael5601** [comment]
I think we will need to switch the grey color based on the backround. There will be no grey tone that fits every backround. Especially when there are more themes involved in the future. 

As soon as we have different color layers set, as discussed in this [issue](https://github.com/eclipse-platform/ui-best-practices/issues/142) it should be very easy to change the colors later on.

**Comment by Michael5601** [comment]
@BeckerWdf Please be also aware that the disconnect icon you mentioned above is "automatically" disabled with the old disablement algorithm in your sreenshot above. Normally the icon is fully red. All other gray icons have a good contrast as they are not disabled but use our specified gray color.

**Comment by jasmin261098** [comment]
Based on this issue, the icon colors were adjusted—among other things to improve contrast. Screenshots can be found in the issue. The work is complete and the issue can be closed.

@BeckerWdf if you are okay with that, please close the issue.
